packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -7,6 +7,102 @@ section: get-started
 import './get-started.css';
 import { Divider, Timestamp } from '@patternfly/react-core';
 
+<Timestamp>March 2024</Timestamp>
+
+## PatternFly 6.2
+
+### Promoted package versions
+- patternfly/patternfly ([changelog](https://github.com/patternfly/patternfly/releases/tag/v6.2.0))
+    - [@patternfly/patternfly@6.2.0](https://www.npmjs.com/package/@patternfly/patternfly)
+- patternfly/react ([changelog](https://github.com/patternfly/patternfly-react/releases/tag/v6.2.0))
+- PatternFly extensions 
+    - [@patternfly/chatbot@2.2.0](https://www.npmjs.com/package/@patternfly/chatbot) ([changelog](https://github.com/patternfly/chatbot/releases/tag/v2.2.0))
+
+### Component updates
+
+#### Bugs 
+- Select: We fixed scrolling for menus with grouped items.
+- Masthead: We correct vertical centering.
+
+#### Features 
+-
+
+### Extension updates 
+
+#### ChatBot
+
+##### Bugs
+- We fixed an overflow issue in popovers with long titles.
+- We fixed the appearance of links.
+- We removed the cancel button from the user feedback form, which was unnecessary due to the presence of a close button.
+- We made resize and form-control overrides more important to assist with downstream implementations.
+- We added loading and error states to the conversation history.
+- We made updates for general style consistency.
+
+##### Features 
+- We added the following new ChatBot features:
+    - [Inline drawer display mode](/patternfly-ai/chatbot/overview/demo/#inline-drawer-chatbot).
+    - [Message feedback and thank-you cards](/patternfly-ai/chatbot/messages#message-feedback).
+    - [Resizable function for the conversation history drawer](/patternfly-ai/chatbot/ui#resizable-drawer).
+    - [Layout for comparing ChatBots](/patternfly-ai/chatbot/overview/demo#comparing-chatbots).
+    - [Support for quick starts](/patternfly-ai/chatbot/messages#messages-with-quick-start-tiles).
+    - [Global ChatBot setting menu](/patternfly-ai/chatbot/ui#settings). 
+    - [Terms of use modal](/patternfly-ai/chatbot/ui#terms-of-use).
+    - Additional message content types (as shown in the [bot messages](/patternfly-ai/chatbot/messages#bot-messages) examples): 
+        - Inline error message.
+        - Block quotes.
+        - Headings.
+        - External links, with styling.
+        - Images.
+        - Tables in messages.
+    - **Note:** We're still updating our design guidelines to reflect these changes and will provide an update once they are complete.
+- New documentation to guide the use of [analytics in ChatBots](/patternfly-ai/chatbot/analytics).
+
```

Comment:
* Added clicked states for response actions and active and selected states for quick responses
* Custom ReactNodes can now be injected in Messages via the `extraContent` prop
* Analytics examples and documentation (unsure if we'd want to call this out here)